setup.py

```diff
@@ -19,7 +19,7 @@
 history = open('CHANGES.rst').read()
 
 tests_require = [
-    'pytest-reana>=0.7.0.dev20191219,<0.8.0',
+    'pytest-reana==0.7.0.dev20191219',
```

Comment:
Doesn't seem good to pin to the old version in case we upgrade pytest-reana for client stuff in the future?!

But actually no REANA package uses today's released pytest-reana 0.7.0.dev20200206 minimal version yet...